### DISCUSSION TITLE
URL to Slideshow?

### DISCUSSION THREAD

**Discussion by rabenimmermehr** [bpatrik/pigallery2#363] [Q&A]
Is there a way to directly launch into the slideshow of a given folder, preferably even with an option to configure the display time of each image?

What I'd like to avoid is to: Open URL of folder -> Click single image -> click on "play"-button in bottom right corner -> open fullscreen.

**Comment by bpatrik** [comment]
I think it is possible to get the url for the slideshow view that opens a particular photo, like: https://pigallery2.herokuapp.com/gallery?p=IMG_9037.jpg

Although, it wont autoplay it. (And it might be a bit buggy if you open a photo that is not among the first few photos)

**Reply by Gdeacon11** [reply]
2 years on from this last reply. Is there now a way to use the URL to go directly into a slide show. I am trying to create a Picture frame using pigallery2 and a chromium client on a pi3B+ and want to go straight to slide show on reboot of the raspberry pi. 

Alternatively, if someone knows where in the code base the "p=IMG_9037.jpg" (in the above example) is parsed in the codebase.I could look at what is possible.

**Comment by bpatrik** [comment]
Tracking bug: https://github.com/bpatrik/pigallery2/issues/809 for this feature request. Lets continue it there.
